hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneAclUtil.java

```diff
@@ -159,6 +165,11 @@ public static boolean checkAclRights(List<OzoneAcl> acls,
    */
   public static boolean inheritDefaultAcls(List<OzoneAcl> acls,
       List<OzoneAcl> parentAcls, OzoneAcl.AclScope scope) {
+    return inheritDefaultAcls(acls::add, parentAcls, scope);
```

Comment:
Was the move away from `addAcl` intentional? `addAcl` handled permission merging, whereas passing `acls::add` here will simply append duplicates. It might be safer to stick with the `addAcl` behavior.